catroid/src/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -216,6 +219,49 @@ public void create() {
 		}
 	}
 
+	public void cloneSpriteAndAddToStage(Sprite cloneMe) {
```

Comment:
Not sure, but maybe synchronize this method, since the stage is also accessing the sprite list !